.changeset/heavy-pens-destroy.md

```diff
@@ -13,9 +13,96 @@
 'ai': major
 ---
 
-chore(embedding-model): add v2 interface
-chore(embedding-model-v2): rename rawResponse to response
-feat(embedding-model-v2): add providerOptions
+## Embedding Model V2
```

Comment:
Given how the `changeset` CLI renders the release notes on GitHub, I would remove the heading of the first line

![image](https://github.com/user-attachments/assets/bf05ab94-df34-4aa9-9dab-58268ae711eb)